src/interface/src/styleguide/treatment-type-icon/treatment-type-icon.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Input } from '@angular/core';
+import { PrescriptionSingleAction } from '../../app/treatments/prescriptions';
+
+@Component({
+  selector: 'sg-treatment-type-icon',
+  standalone: true,
+  imports: [],
+  templateUrl: './treatment-type-icon.component.html',
+  styleUrl: './treatment-type-icon.component.scss',
+})
+export class TreatmentTypeIconComponent {
+  @Input() treatment: PrescriptionSingleAction | null = null;
+
+  readonly treatmentIcons: Record<PrescriptionSingleAction, string> = {
+    NO_TREATMENT: 'treatment_none.svg',
+    MODERATE_THINNING_BIOMASS: 'treatment_blue.svg',
+    HEAVY_THINNING_BIOMASS: 'treatment_purple.svg',
+    MODERATE_THINNING_BURN: 'treatment_orange.svg',
+    HEAVY_THINNING_BURN: 'treatment_yellow.svg',
+    MODERATE_MASTICATION: 'treatment_junglegreen.svg',
+    HEAVY_MASTICATION: 'treatment_limegreen.svg',
+    RX_FIRE: 'treatment_red.svg',
+    HEAVY_THINNING_RX_FIRE: 'treatment_brown.svg',
+    MASTICATION_RX_FIRE: 'treatment_pink.svg',
+  };
```

Comment:
Since this is only used within this component, I think it's reasonable to keep this Record here, as long as the Prescription type is elsewhere.